### PR TITLE
fix(project): Fix HTML, give better semantics and legibility

### PR DESCRIPTION
**Pull request by lasjorg**
#### Pre-Submission Checklist

- [x] Your pull request targets the `master` branch of freeCodeCamp/testable-projects-fcc
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] Your changes have been tested either locally or using a newly created CDN based on your fork's testable-projects-fcc/build/bundle.js file
- [ ] Working changes have been added to the build by running `npm run build`

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.

- [ ] Closes currently open issue: Closes #XXX

#### Description

1. Move the h3 out of the ul

2. Change the #img-div to a figure element, change the #img-caption to a figcaption element. As per https://github.com/freeCodeCamp/testable-projects-fcc/issues/759, I did not change the id name to avoid breaking backward compatibility. Let me know if this is _not_ acceptable.

3. New system font stack, better font sizing, and line height.

4. Added cite element and cite attribute.

5. Added some media queries. I know campers are not required to use media queries on this project, but this just gives some super simple example media queries.

I have tried my best to keep the look of the original with just some small subtle visual changes.

All "Tribute Page" tests are passing.
HTML and CSS passes validation tests.

### CODE REVIEW

**Comment by tbushman** [comment]
@lasjorg Initially I understood that you wanted to change `id`s, but now it's more clear. In order to implement this, we would also need to coordinate re-wording the user stories everywhere they are written -- in the tests here, and on freecodecamp.com. I'll leave this open for discussion.

**Comment by lasjorg** [comment]
I'm willing to help change the project description text and test. It doesn't look that bad. Although I'm not sure about the deployment part, or if there are other issues I'm not aware of.

If these changes are something the maintainers of the repo, or whoever makes these decisions, find worthwhile, then I would also suggest changing the id="main" div to the main element and id="tribute-info" div to the section element while we're at it. The HTML semantics of this project suffers from "do as I say, not as I do". Just look at the image alt text (which I totally forgot to change), "pic from flickr", really?

I understand there is a lot of potential for breakage here, and it's not something to be taken lightly. Anyway, in my opinion, at some point these projects will have to be updated.

For now, I can revert the elements and just keep all non-breaking changes.

**Comment by tbushman** [comment]
@lasjorg It's possible to do, and so I'll just give some info: You would want to open an issue at github.com/freeCodeCamp/freeCodeCamp with the request to change the user story to accommodate the tags. If the change isn't made in the user story, then we wouldn't need this change, and vice versa.  

If you want this functionality in your own project, while passing tests, please submit code changes of `src/project-tests/tribute-page-tests.js`, in particular beginning line 51, in this repository. You could create a regex or similar to allow these tags. Let us know if you need any other info.

**Comment by lasjorg** [comment]
Does the test have to be backward compatible with projects made with the old requirements? Looking at the test I'm not sure how I would do that.

**Comment by lasjorg** [comment]
I have little experience with Chai/Mocha, so I might be wrong. But, it seems to me I have to conditionally run the tests and for that, I need to know which test the user is trying to pass. I can't see a reliable way of doing that.

**Comment by tbushman** [comment]
@lasjorg I think the original aim was to change the tests to allow use of these `figure` and `figcaption` tags? If so, you would submit additional changes to the tests. Or, if the aim is to clean up the example project, and you're ok with not changing the `img-div` and `img-caption` element tags, that would probably pass the current tests.

**Comment by lasjorg** [comment]
@tbushman I do want to change the elements, I'm just not sure if what I have for the test is acceptable. I will make a PR for the test.

Also, it seems like the CI build (on this PR) will fail until the new test is merged into master, right?

File: src/projects/tribute-page/index.html [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+  <p>The man who saved a billion lives</p>
+  <figure id="figure">
     <img
       id="image"
       src="https://c2.staticflickr.com/4/3689/10613180113_fdf7bcd316_b.jpg"
-      alt="pic from flickr"
+      alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists"
     />
-    <div id="img-caption">
+    <figcaption id="figcaption">
```
Comment (tbushman) [inline src/projects/tribute-page/index.html]: Change this to `<figcaption id="img-caption">` and the above to `<figure id="img-div">` and see if that passes the tests. The css declarations would need to change accordingly.

**Comment by tbushman** [comment]
@lasjorg I'll comment where I think these tests are failing. The main concern is the wording in the user stories, I'm realizing now, since the tests ignore the tag type. If the user story says `<div>` and the example project uses `<figure>` , do you think that's an issue?

File: src/projects/tribute-page/index.html (hunk 1 above)
Comment (lasjorg) [reply src/projects/tribute-page/index.html]: That is what I did, to begin with. That's how the initial PR was.

**Comment by lasjorg** [comment]
> @lasjorg I'll comment where I think these tests are failing. The main concern is the wording in the user stories, I'm realizing now, since the tests ignore the tag type. If the user story says `<div>` and the example project uses `<figure>` , do you think that's an issue?

I do think it's a problem, and it is why I said I would need to do conditional testing. The assertion tests won't be correct otherwise. Let me do the PR for the new test and we can see if it's an acceptable solution or not, maybe @ValeraS can weigh in on the test as well.

**Comment by ValeraS** [comment]
Done in #781

**Pull request closed without merging**